tests/test_worker.py

```diff
@@ -347,6 +347,16 @@ def test_custom_job_class(self):
         worker = Worker([q], job_class=CustomJob)
         self.assertEqual(worker.job_class, CustomJob)
 
+    def test_custom_job_class_is_not_global(self):
```

Comment:
Never mind, this makes sense if you take https://github.com/nvie/rq/pull/657/files into account. We'll keep this here :).